portfolio/src/main/webapp/script.js

```diff
@@ -16,17 +16,31 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['I have lived in the same house my whole life!', 
-      'I hate tomatoes!', 'I do aerial silks!', 
-      'I studied abroad twice!', 'I took the picture above in New Zealand!', 
-      'I went bunjee jumping and skydiving on the same day!',
-      'The picture to the right is from Barcelona!'];
+  const greetings = [
+    "I have lived in the same house my whole life!",
+    "I hate tomatoes!",
+    "I do aerial silks!",
+    "I studied abroad twice!",
+    "I took the picture above in New Zealand!",
+    "I went bunjee jumping and skydiving on the same day!",
+    "The picture to the right is from Barcelona!",
+  ];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
+  const greetingContainer = document.getElementById("greeting-container");
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches data from /data and displays it as html on index page.
+ */
+function fetchContent() {
+  fetch("/data")
+    .then((response) => response.text())
+    .then((content) => {
+      document.getElementById("content-container").innerHTML = content;
+    });
+}
```

Comment:
Can you try using `async`/`await` here like:

```js
async function fetchContent() {
  const response = await fetch('/data');
  document.getElementById('content-container').innerHTML = response.text();
}
```

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await